src/index.json

```diff
@@ -1377,6 +1377,46 @@
                     "version": "0.4.2"
                 },
                 "sha256Digest": "556ce59da6989c1454393deffb6f00887dfd9debaccc2ea9dbb393057e34540b"
+            },
+            {
+                "downloadUrl": "https://azurecliprod.blob.core.windows.net/cli-extensions/interactive-0.4.3-py2.py3-none-any.whl",
+                "filename": "interactive-0.4.3-py2.py3-none-any.whl",
```

Comment:
Since this and the entry for 0.4.2 have the same minCliCoreVersion, go ahead and remove that entry.